src/components/HeaderMenu/HeaderMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Layout, Menu, Typography, Button, theme } from 'antd';
+import { ItemType } from 'antd/lib/menu/hooks/useItems';
+import { NavLink } from 'react-router-dom';
+import styles from './HeaderMenu.module.scss';
+import routePaths from '../../utils/constants/routePaths';
```

Comment:
We need to setup an alias in the vite config to avoid these long dots imports -> import routePaths from '@/utils/constants/routePaths'; - for example. (@ -> src folder)